src/test/java/healthy/lifestyle/backend/users/controller/AuthControllerTest.java

```diff
@@ -77,7 +78,8 @@ void postgresqlContainerTest() {
     void signupTest_shouldReturn201Created() throws Exception {
         // Given
         Role role = dataHelper.createRole("ROLE_USER");
-        SignupRequestDto signupRequestDto = dataUtil.createSignupRequestDto("one");
+        Country country = dataHelper.createCountry(1);
+        SignupRequestDto signupRequestDto = dataUtil.createSignupRequestDto("one", country.getId());
```

Comment:
Let's ensure that user has been created in a database and associated with a proper country. Maybe add `DataHelper.getUserById()` and use this method here to verify a new user and country?